### PR TITLE
📝 add error documentation

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

**Review by jerembdn — APPROVED** [review]

**Review by AntoineKM — CHANGES_REQUESTED** [review]
fix it:
inline-block:
image

inline-flex:
image

File: docs/pages/docs/components/error.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  </Container>
+</Container>
+`}
+/>
+
+## With an `error` property
+
+<Playground
+  code={`
+  <Error error={{ message: 'The request failed.', action: 'Contact Us', link: 'https://vercel.com/contact' }}/>
```
Comment (AntoineKM) [inline docs/pages/docs/components/error.mdx:84]: indent
```
<Error 
  error={{
    message: 'The request failed.',
    action: 'Contact Us',
    link: 'https://vercel.com/contact' 
  }}
/>
```

**Review by AntoineKM — CHANGES_REQUESTED** [review]
All okay but add a changeset

**Comment by AntoineKM** [comment]
also use double quotes

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**